consts/user-roles.js

```diff
@@ -0,0 +1,9 @@
+const USER_ROLES = {
+  ROLE_USER: 'user',
+  ROLE_ADMIN: 'admin',
+  ROLE_SUPERADMIN: 'superadmin',
+};
```

Comment:
we already have roles in consts/index.js

Reply:
I deleted it and replaced it with roles from consts/index.js. But in user-model.js, I imported roles like this:
```
const {
  roles,  //instead of roles: {USER, ADMIN, SUPERADMIN}
} = require('../../consts');
```

...because there is **another const USER** from DB_COLLECTION_NAMES

Reply:
So in the code, I use roles like this: 


```
 role: {
    type: String,
    enum: [roles.USER, roles.ADMIN, roles.SUPERADMIN],
    default: roles.USER,
  },
```
is it ok?